web.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from twitter.stream import loop
+app = Flask(__name__)
+
```

Comment:
What is the purpose of this flask? Will it serve to show the tweets in a map?

Reply:
I was thinking to build a basic flask application but I thing this is an is for another pull requests, so I ll remove it

Reply:
Cool, ty!